content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

```diff
@@ -113,11 +138,38 @@ MTM-42808</td>
 
 <tr>
 <td>
-Rest API</td>
+REST API</td>
 <td>The <code>detail()</code> and <code>delete()</code> method of the @c8y/client MeasurementService has been marked as deprecated as it is not supported with the new timeseries.</td>
 <td>
 MTM-48529</td>
 
+<tr>
+<td>REST API</td>
+<td>The `activeVersionId` and the hosted application's manifest will now be updated when updating the `latest` version of an application.</td>
+<td>MTM-48430</td>
+</tr>
+
+<tr>
+<td>REST API</td>
+<td>The REST method <code>GET /measurement/measurements/{id}</code> is no longer supported for timeseries measurements.</td>
+<td>MTM-48344</td>
+</tr>
+
+<tr>
+<td>Security</td>
+<td>An attribute was added to all external application links, that prevents tab nabbing (a security vulnerability that could be used to gain the user's browser session).</td>
+<td>MTM-48156</td>
+</tr>
+
+<tr>
+<td>Single Sign On</td>
+<td>The single sign-on (SSO) configuration page has been extended by a new access mapping option. Instead of a checkbox which sets the mapping during each login or only during user creation, a radio button has been added. Now three options are available:
+<br>- Perform access mapping for every login for all privileges (admin can not edit SSO user privileges)
+<br>- Perform access mapping only on user creation (admin can edit SSO user privileges)  
+<br>- Perform access mapping for every login for those privileges listed in the configuration (admin can edit SSO user privileges)</td>

```

Comment:
there's no periods at the end of the list items. Intended?

Reply:
I think for release notes purposes this is ok, but I could add them?